docs/features/refout.md

```diff
@@ -67,10 +68,10 @@ As mentioned above, there may be further refinements after C# 7.1:
 - produce ref assemblies even when there are errors outside method bodies (emitting error types when `EmitOptions.TolerateErrors` is set)
 
 ## Open questions
-- should explicit method implementations be included in ref assemblies?
+- should explicit method implementations be included in ref assemblies? (answer: no. The interfaces that are declared as implemented are what matter to consuming compilations)
```

Comment:
@gafter may want to comment. From our discussion, it was ok to strip explicit method implementations.

That said, there is still a PROTOTYPE marker in this branch for this (see `GetExplicitImplementationOverrides`).